spec/integration/elasticsearch/spec_helper.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+require 'rake'
+require 'elasticsearch'
+require 'elasticsearch/extensions/test/cluster/tasks'
+require 'faraday/error'
+require 'uri'
+
+ELASTIC_ARGS = {
+  network_host: URI(Config.elastic_url).host,
+  port: URI(Config.elastic_url).port.to_s,
+  nodes: 1,
+  multicast_enabled: false
+}
+has_started_elastic = false
+
+def elastic_running?
```

Comment:
This, along all those `before` contexts etc. should go to a separate support file.